models/issue_list.go

```diff
@@ -429,6 +429,7 @@ func (issues IssueList) loadTotalTrackedTimes(e Engine) (err error) {
 
 		// select issue_id, sum(time) from tracked_time where issue_id in (<issue ids in current page>) group by issue_id
 		rows, err := e.Table("tracked_time").
+			Where("deleted is false").
```

Comment:
I'm suspicious that this is incorrect... Shouldn't this be:

```suggestion
			Where("deleted = 0").
```

Reply:
@lunny @zeripath  I dont know how xorm handle bool for each db engine?